CHANGELOG.md

```diff
@@ -10,7 +10,16 @@ The format is based on [Keep a Changelog], and this project adheres to
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/envato/stack_master/compare/v2.13.3...HEAD
+[Unreleased]: https://github.com/envato/stack_master/compare/v2.13.4...HEAD
+
+## [2.13.4] - 2023-07-31
+
+### Fixed
+
+- Resolve error caused by `SortedSet` class being removed from the `set` library ([#374]).
```

Comment:
This is only a problem for those projects using SparkleFormation.

```suggestion
- Resolve SparkleFormation template error caused by `SortedSet` class being removed from the `set` library in Ruby 3 ([#374]).
```

Reply:
Thanks, fixed in [254eb03](https://github.com/envato/stack_master/pull/374/commits/254eb03871efc652a46fb542f15514c0e76a8b67).